### PR TITLE
Reword some of the existing VCS notes in preparation for new ones

### PR DESCRIPTION
**Pull request by tchebb**
- Make "us" / "you" wording consistent with CLI notes.
- Clarify some awkward passages.
- Improve formatting.

### CODE REVIEW

File: lecture-notes/2-vcs.md [hunk 1]
```diff
@@ -257,8 +260,7 @@ Changes to be committed:
 $
 ```
 
-This has now changed your `.git/` directory! Take a look. You should see
-something like this:
+After running this, you'll finally see a change in your `.git/` directory!
```
Comment (tekknolagi) [inline lecture-notes/2-vcs.md:263]: I think "something like this" is important -- it may not be exact

File: lecture-notes/2-vcs.md [hunk 2]
```diff
@@ -238,13 +240,14 @@ untracked file named `myfile`. From Git's point of view, untracked files don't
 exist: it will tell you about them in `git status`, but it won't include them
 in commits, meaning it won't track their history.
 
-To verify this, take a look at the `.git/` directory again and observe nothing
-has changed: no new objects are present, nor have any of the files--`config`,
-`description`, `HEAD`, or `info/exclude`--been altered.
+To verify this, take a look at the `.git/` directory again and observe that
+nothing has changed: no new objects are present, nor have any of the
+files--`config`, `description`, `HEAD`, or `info/exclude`--been altered.
```
Comment (tekknolagi) [inline lecture-notes/2-vcs.md:245]: nit for not-this-PR: we should be using `---` (three)

File: lecture-notes/2-vcs.md [hunk 3]
```diff
@@ -143,11 +145,11 @@ Each subcommand is basically its own command--they just all happen to be
 implemented inside a single program, `git`. Each subcommand has its own man
 page, whose name is prefixed with `git-`. For example, `man git-init`.
 
-Let's try out the `git status` subcommand on a new repository! `git status`
-tells you what Git thinks the current state of things is. It shows you a
-helpful English description of what's going on, as well as some suggested
-things to do next. In this case, it helpfully suggests that you make and
-*track* some files:
+Try out the `git status` subcommand! `git status` tells you what Git thinks the
+current state of things is. It shows you a helpful English description of
```
Comment (tekknolagi) [inline lecture-notes/2-vcs.md:149]: s/English/prose, maybe, for localization?

File: lecture-notes/2-vcs.md [hunk 4]
```diff
@@ -123,12 +125,12 @@ hood, it represents old versions of your files, as well as changes to those
...
-point in time: it holds a list of files, the contents of those files, the date
-at which it was created, who created it, and a user-provided description of
-what changed since the previous commit, which it also stores a reference to.
-We'll talk more about commits shortly.
+There are several types of object, but the only one you'll generally interact
+directly with is called a *commit*. A commit represents the state of a
+repository at some point in time: it holds a list of files, the contents of
+those files, the date at which it was created, who created it, and a
+user-provided description of what changed since the previous commit, which it
+also stores a reference to.  We'll talk more about commits shortly.
```
Comment (tekknolagi) [inline lecture-notes/2-vcs.md:133]: nit: one space between sentences

**Review by tekknolagi — APPROVED** [review]

File: lecture-notes/2-vcs.md (hunk 1 above)
Comment (tchebb) [reply lecture-notes/2-vcs.md:263]: The hash should be exact, right? It's a blob, so its hash won't depend on date, author, or anything that would differ.

Comment (tekknolagi) [reply lecture-notes/2-vcs.md:263]: might have different hash function or idk

**Pull request merged**